superdesk/commands/index_from_mongo.py

```diff
@@ -89,7 +106,11 @@ def get_mongo_items(cls, mongo_collection_name, page_size, last_id, string_id):
         bucket_size = int(page_size) if page_size else cls.default_page_size
         print("Indexing data from mongo/{} to elastic/{}".format(mongo_collection_name, mongo_collection_name))
 
-        db = app.data.get_mongo_collection(mongo_collection_name)
+        try:
+            db = get_current_app().mongo.get_collection(mongo_collection_name)
+        except KeyError:
+            db = app.data.get_mongo_collection(mongo_collection_name)
```

Comment:
These `try/except` would only be in these low level parts of the async app. Most devs would not need to use the mongo/elastic clients directly, but rather through a resource service like we do now.

This will be in my next PR, which allows a dev to interact with resources without knowing the underlying data layer(s).
```python
from my_module.users import UsersService, User

async def test_users():
    service = UsersService()  # automatic singleton instance
    user = User(id="user_1", first_name="Monkey", last_name="Magic")
    await service.create([user])
    my_user = await service.get_by_id("user_1")
    ...
```

Reply:
got it!